### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by rushingMarina**

### CODE REVIEW

**Comment by rushingMarina** [comment]
@shestakoven Can we merge this?

**Review by shestakoven — COMMENTED** [review]
Each json can represents a group of tags. Stable coins for example, there is two tags

File: readme.md [hunk 1]
```diff
@@ -1 +1,26 @@
-Tags
+## Overview
+DexGuru uses token lists in this repository to group tokens by tags. Check out the visual representation at https://dex.guru/tokens. 
+Each JSON file represents a single tag and is a list of publicly sourced tokens. Anyone is welcome to add new tokens and new tags.
```
Comment (shestakoven) [inline readme.md:3]: It can be a group of tags

Comment (sneg55) [reply readme.md]: thanks, I'll update this

**Pull request merged**